src/components/Badge/docs/BadgeExtendedExample.st.css

```diff
@@ -1,12 +1,19 @@
 :import {
     -st-from: "../Badge.st.css";
     -st-default: TPABadge;
+    -st-named: badgeOverrides;
 }
 
-.root {
+.mixAll {
     -st-mixin: TPABadge(
         BadgeBgColor '"color(--badgeBgColor)"',
         BadgeTextColor '"color(--badgeTextColor)"',
-        BadgeBorderColor '"color(--badgeBorderColor)"'
     );
-}
\ No newline at end of file
+}
+
+.mixOverrides {
+    -st-mixin: badgeOverrides(
```

Comment:
@jonathanadler - what do you think of the API? Think that this should become some kind of a convention for all the components we will create